### ISSUE TITLE
As a front end user, I want to see unpublished scholarship records displayed in CMS citation format so that users can easily cite unpublished material available on the PGP.

### ISSUE THREAD

**Issue by kseniaryzhova** [Princeton-CDH/geniza#1502]
### testing notes (QA)

On the [QA public site](https://test-geniza.cdh.princeton.edu/), check some documents with unpublished scholarship records, and navigate to the bibliography tab.
- [x] Goitein index cards should be formatted like "S. D. Goitein, unpublished index cards (1950–85), #5546, #8808, #8809 and #8810. Princeton Geniza Lab, Princeton University."
- [x] Other unpublished Goitein records should be formatted like "S. D. Goitein's unpublished edition (1950–85)" if they are not digitized, or "S. D. Goitein's unpublished edition (1950–85), available online through the Princeton Geniza Project at [url]" if they are.
- [x] Non-Goitein unpublished records should be formatted like "Yusuf Umrethwala's digital edition (2023), available online through the Princeton Geniza Project at [url]," or "A. L. Udovitch and Marina Rustow's digital translation, available online through the Princeton Geniza Project at [url]." They should be accurate about the kind of scholarship they include, i.e. edition vs translation should be correctly identified.

In the transcription/translation viewer, for records that have digitized transcriptions and translations, you should see the shorter versions of those citations right above the transcription/translation text, like so:
- [x] Goitein unpublished scholarship: "S. D. Goitein's unpublished edition (1950–85)."
- [x] Non-Goitein unpublished scholarship: "Yusuf Umrethwala's digital translation (2023)," or "A. L. Udovitch and Marina Rustow's digital translation."

In the QA admin site, navigate to some documents with unpublished digital editions or translations.
- [x] On a footnote for an unpublished digital edition/translation, fill out the "with minor emendations by" field with appropriately formatted information (i.e. Person Name, Year). When you navigate back to the public document detail page for this document, and scroll down to the transcription/translation viewer, this emendation information should appear in the citation.
- [x] The emendation information you added should also appear on the bibliography tab.

Next, go to the [published Geniza production site](https://geniza.princeton.edu/). 
- [x] Compare all of the above records to the versions on production. Confirm no important information is missing from the new citation.

(Note that the formatting has undergone some revisions since the initial description was written.)

---

**Describe the solution you'd like**
There are two major types on unpublished records cited on the PGP: S.D. Goitein's work (transcriptions, translations, index cards) and everyone else's (modern) transcriptions and translations. For the latter, since it's all modern, we have the authors themselves input their dates. For Goitein's unpublished works we instead have a date range we would like to display on the public site (it will not be displayed on the admin interface for the sources). 

**How to cite Goitein's notecards:**
Goitein, S.D. [Notecard #] [Link to notecard], (1950–85), unpublished notecards. Princeton Geniza Lab, Princeton University.

**How to cite unpublished translations and transcriptions (Goitein examples also included):**
[Library,][Shelfmark]. [Author's digital edition and/or translation], [optional: with minor emendations by other author (date)], available online through the Princeton Geniza Project at [link to doc page] [(accessed [date])].

**For Goitein's editions and translations that are unpublished and undigitized, please display "unpublished edition and/or translation."**

Examples:

Cambridge University Library, T-S 8J17.15. S. D. Goitein’s unpublished edition (1950–85) available online through the Princeton Geniza Project at https://geniza.princeton.edu/documents/7665/ (accessed June 12, 2023).

Cambridge University Library, T-S 12.373. S. D. Goitein’s unpublished edition (1950–85), with minor emendations by Alan Elbaum (2021), available through the Princeton Geniza Project at https://geniza.princeton.edu/documents/3146/ (accessed April 20, 2023).

Jewish Theological Seminary, ENA NS 16.28. Alan Elbaum and Yusuf Umrethwala's digital edition and translation (2022) available online through the Princeton Geniza Project at https://geniza.princeton.edu/documents/11714/ (accessed May 1, 2022).

**Comment by kseniaryzhova** [comment]
@blms are we waiting on Chelsea's document detail page designs for this?

**Comment by blms** [comment]
@kseniaryzhova Nope! I can work on this anytime—how high priority?

**Comment by blms** [comment]
Note from today's meeting: we decided that in cases where the footnote is on an entity like a person or place, the doc relation is always discussion; therefore, when citing unpublished sources for these (in the rare case where it happens), it should read "discussion".

**Comment by blms** [comment]
Lots of questions on this one as I begin work on it!

1. The display of library and shelfmark, and the access date for PGP, are both new to the system; we haven't displayed those in scholarship records page previously. Should those actually show up on the scholarship records page? Or are those only meant for [How To Cite](http://localhost:8000/en/about/how-to-cite/)?
   - If library/shelfmark/access date are only supposed to be included in How To Cite, would the examples look the same for the Scholarship Records page but just without those?
   - If they are meant to display even on the scholarship records page, how should we handle joins where the parts come from different libraries? I'm already seeing this get out of hand, like on PGPID 4182:
   ![Screenshot 2024-03-11 at 3 48 11 PM](https://github.com/Princeton-CDH/geniza/assets/7234006/cd687c02-6124-4a96-a509-9876431a9d3a)
2. There are a few other places where we also display citations:
    - [Document details page](https://geniza.princeton.edu/en/documents/11714/), top metadata section under Editors
    - Document details page, transcription/translation viewer: on the dropdown to choose which edition or translation you are viewing
    - Document details page, underneath the header Transcription or Translation, depending on which is being viewed
  
   How are these affected by the changes? And are they affected differently from each other?
3. Goitein: Are we going with "Notecard" instead of "index card"? In the DB they're all listed as "index cards".
4. Goitein: I noticed the formatting for Goitein notecards are listed last name, first name, but everywhere else on PGP we use first then last ([example](https://geniza.princeton.edu/en/documents/5528/scholarship/)). Should we be consistent and stick to first then last here?

**Comment by kseniaryzhova** [comment]
Here is how we would like for the unpublished citations to appear:

**How to cite Goitein's notecards:**
S.D. Goitein, [index card #] [Link to card], (1950–85), unpublished index cards, available online through the Princeton Geniza Project at [link].

(We will have a separate line on the How to Cite page indicating the physical index cards are at the NLI. RSK approved.)

**How to cite unpublished translations and transcriptions (Goitein examples also included):**
[Library,][Shelfmark]. [Author's digital edition and/or translation], [optional: with minor emendations by other author (date)], available online through the Princeton Geniza Project at [link to doc page] [(accessed [date])].

Note: Library, Shelfmark and accessed date will ONLY be visible on the bottom of the doc detail page where the permalink is currently. Scholarship records will not have library, shelfmark and accessed date (but will have everything else). The same citation will appear as the header of the transcription/translation editor when that transcription/translation is selected.

Source selection format will remain the same. And the dropdown of the translation/transcription module, when one is selecting a source to view, will be a pared down version of the scholarship record (Last name, a few keywords of the title or the whole title if it's short, doc number if applicable and date of publication)

**Comment by blms** [comment]
Note from meeting: if the notes field starts with capital W "with", just lowercase it.

**Comment by blms** [comment]
Question label here is because we are still waiting on determination about "with emendations by"—should address in meeting today

**Comment by richmanrachel** [comment]
@blms - did we end up answering your question about "emendations by"? I can also add this completion of formatting citations to issue #1686

**Comment by blms** [comment]
@richmanrachel I think so! I gather from the meeting this one is paused until after #1686 is completed, so that we can include emendations in the citations, which is currently "on hold" per meeting notes.

Another option would be to proceed with this issue and just exclude emendations from the citations initially, until #1686 and then as you say, include updating the citations in that issue.

**Comment by kseniaryzhova** [comment]
@blms Is there a way to ignore the period if someone adds that to the emendations field? The help text implies you need one, and I think some people will just naturally put one there, but it comes on wonky in public view: 

![Image](https://github.com/user-attachments/assets/e594c3de-82a9-41f6-a418-49f0d3b64aed)

**Comment by kseniaryzhova** [comment]
@blms Looks good, and I confirmed with the team this is the language/format that we want for these unpublished citations! Closing, thank you!
